dev-haskell/mysql/mysql-0.1.7.2-r1.ebuild

```diff
@@ -20,7 +20,8 @@ IUSE=""
 
 RESTRICT=test # needs local mysql
 
-RDEPEND=">=dev-haskell/semigroups-0.11:=[profile?] <dev-haskell/semigroups-0.19:=[profile?]
+RDEPEND="dev-db/mysql:=
```

Comment:
Reminder in case either of us forget: do _not_ depend on `dev-db/mysql`. From documentation, `dev-db/mysql-connector-c` probably suffices.

Reply:
I think I'll take this commit out entirely and rather deal with it in #1128